public/app/plugins/datasource/elasticsearch/datasource.ts

```diff
@@ -936,49 +936,49 @@ export class ElasticDatasource
  * Exported for tests.
  */
 export function enhanceDataFrame(dataFrame: DataFrame, dataLinks: DataLinkConfig[], limit?: number) {
-  const dataSourceSrv = getDataSourceSrv();
-
   if (limit) {
     dataFrame.meta = {
       ...dataFrame.meta,
       limit,
     };
   }
 
-  if (!dataLinks.length) {
+  if (!dataLinks.length || !dataLinks) {
```

Comment:
Is it needed? I think `dataLinks` are always defined (initialized in the [constructor](https://github.com/grafana/grafana/blob/main/public/app/plugins/datasource/elasticsearch/datasource.ts#L113))

Reply:
i have no idea why i added it 🤔